source/include-link.txt

```diff
@@ -0,0 +1,209 @@
+.. _cpp-include-link:
+
+===========================================
+Include and Link the Driver in Your Program
+===========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+Overview
+--------
+
+In this guide you will learn how to use the {+driver-short+} in your own projects. Examples are provided for using the {+driver-short+} with CMake and with ``pkg-config``.
+
+Example Code
+~~~~~~~~~~~~
+
+The examples on this page use the ``view_and_value.cpp`` example program from the {+driver-short+} source code. Obtain the {+driver-short+} source code by following the :ref:`cpp-quick-start-download-and-install` guide, if you have not previously done so.
+
+You do not need to run this program on a MongoDB server.
+
+Library References
+~~~~~~~~~~~~~~~~~~
+
+The examples use references to components of the {+driver-short+}. For CMake projects, the ``mongo::bsoncxx_shared`` target is used. For pkg-config projects, the ``libbsoncxx`` package is used. You may use the following library targets, depending on the needs of your project:
+
+- CMake
+  - ``mongo::bsoncxx_shared``
+  - ``mongo::mongoccxx_shared``
+  - ``mongo::bsoncxx_static``
+  - ``mongo::mongoccxx_static``
+- pkg-config
+  - ``libbsoncxx``
+  - ``libmongocxx``
+  - ``libbsoncxx-static``
+  - ``libmongocxx-static``
+
+The availability of targets is dependent on the particular installation method.
+
+In the sections that follow, replace ``<path-to-mongo-cxx-driver-sources>`` with the actual path where the {+driver-short+} source tree is located on your system.
+
+CMake
+-----
+
+You can use the {+driver-short+} through CMake. CMake provides the ``find_package`` command, which your project can use to locate the {+driver-short+} once it is installed. Alternatively, your project can use the advanced ``add_subdirectory`` command without requiring installation of the {+driver-short+}.
+
+With Driver Installation
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Installing the {+driver-short+} enables using it in a simple and straighforward way. After installing, you can use CMake's ``find_package`` command to integrate the {+driver-short+} with your project. To learn how to install the {+driver-short+}, visit the following guides:
+
+- :ref:`cpp-quick-start-download-and-install` to install from source
+- :ref:`cpp-installation-advanced` to install from packages
```

Comment:
S: the first sentence uses some subjective language; maybe just remove it.

Also, I think a ..tip admonition would be useful for the information about visiting installation guides. We often use tips when linking out to other relevant pages.

```suggestion
After installing the {+driver-short+}, you can use CMake's ``find_package`` command to integrate the driver with your project.

.. tip::

   To learn how to install the {+driver-short+}, visit 
   the following guides:

   - :ref:`cpp-quick-start-download-and-install` to 
      install from source
   - :ref:`cpp-installation-advanced` to install from 
      packages
```